PDFtoWord/main.py

```diff
@@ -0,0 +1,9 @@
+from pdf2docx import Converter
+
+pdf_file = 'sample.pdf'
+docx_file = 'sample.docx'
```

Comment:
Prompt the user whether they want to define a custom file name for the doc. If they type "Y", then take the input for the filename. If "N" then the doc file name should be the same as the pdf file added (doc). For example:

input file: abc.pdf
Do you want to provide a custom name for the word file? (Y/N):
if "Y": 
  enter the file name without extension: 
elif "N":
 output file name: abc (doc).docx

Reply:
Okay